server/app/repository/SearchParameters.java

```diff
@@ -0,0 +1,25 @@
+package repository;
+
+import com.google.auto.value.AutoValue;
+import java.util.Optional;
+
+@AutoValue
+public abstract class SearchParameters {
```

Comment:
SearchParameters is pretty generic name. Given that this class works specifically with search parameters for clients in TI. I can think of few options:
1. Rename it to TrustedIntermediaryClientSearchParams (long :/)
2. Introduce `ti` package and move it there as ClientSearchParams.

@clouser2 @swatkat1 are there precedents for passing search data around for other usecases?

Reply:
Thank you for you feedback. But I am taking this in the second iteration.

Reply:
+1 that we likely want a more specific name, such as TIAccountFilters or something along those lines.

Mind leaving a TODO to rename this to something different so it can be tracked against the larger issue for cleanups?

Reply:
suggestion (blocking): please change this name and document the class before merging. It is very generic and likely to cause confusion.